src/runtime_src/core/common/info_memory.cpp

```diff
@@ -467,8 +467,8 @@ scheduler_update_stat(const xrt_core::device *device)
   try {
     // lock xclbin
     std::string xclbin_uuid = xrt_core::device_query<xq::xclbin_uuid>(dev);
-    // dont open a context if xclbin_uuid is empty
-    if(xclbin_uuid.empty())
+    // dont open a context if xclbin_uuid is empty or is all zeros
+    if (xclbin_uuid.empty() || !xrt::uuid(xclbin_uuid))
```

Comment:
What was this change required?

Reply:
There was an bug on the dynamic report page that would cause a crash due to the xclbin uuid being all 0s. Since the uuid is not "empty" the software in C++ and Python would attempt to open context on a non existent file which caused strange issues. C++ it would crash when opening the context, in Python is would crash when closing which failed to catch for one reason or another.